src/js/components/PodInstancesTable.js

```diff
@@ -281,6 +290,31 @@ class PodInstancesTable extends React.Component {
     );
   }
 
+  renderColumnLogs(prop, row, rowOptions = {}) {
+    if (rowOptions.isParent) {
+      return <Icon id="" size="mini" />;
```

Comment:
As discussed with orlando, just make sure to add a comment for clearly stating why we are using nbsp there.